apps/configurator/lib/configurator_web/controllers/consumable_item_controller.ex

```diff
@@ -28,7 +29,8 @@ defmodule ConfiguratorWeb.ConsumableItemController do
 
   def show(conn, %{"id" => id}) do
     consumable_item = Items.get_consumable_item!(id)
-    render(conn, :show, consumable_item: consumable_item)
+    version = Configuration.get_version!(consumable_item.version_id)
```

Comment:
Why not preloading these in the `get_consumable_item/1` function?

Reply:
I didn't see much worth preloading the version, changing would imply a refactor in the views and test. I would do it in the next PR

Reply:
Sounds good!